worlds/mmbn3/docs/setup_en.md

```diff
@@ -18,11 +18,12 @@ on Steam, you can obtain a copy of this ROM from the game's files, see instructi
 
 Once Bizhawk has been installed, open Bizhawk and change the following settings:
 
-- Go to Config > Customize. Switch to the Advanced tab, then switch the Lua Core from "NLua+KopiLua" to
+- **If you are using a version of BizHawk older than 2.9**, you will need to modify the Lua Core.
+  Go to Config > Customize. Switch to the Advanced tab, then switch the Lua Core from "NLua+KopiLua" to
   "Lua+LuaInterface". This is required for the Lua script to function correctly.
```

Comment:
I'm unsure, but it might have been intended for **Note:** to be at the start of a new line, in which case you want these spaces. Potentially the same for the note under "Connect to the Multiserver" (line 78)
```suggestion
  "Lua+LuaInterface". This is required for the Lua script to function correctly.  
```

Reply:
Good catch, I've made both NOTEs capitalized and on new lines.